### PR TITLE
DEV-34 replace relative path for static files on absolute; changes routes

### PR DESCRIPTION
**Pull request by NikolayYakovenko**
There was a simple problem with path for static files. You should use an absolute path for it.
https://github.com/ReactTraining/react-router/blob/master/FAQ.md#why-doesnt-my-application-work-when-loading-nested-routes

At first, I thought that problem connected with react-router, but I was wrong.
Also I've changed **routes.js**, because, as I know, the app should has one **Router** and different **Route** to render UI.

### CODE REVIEW

**Comment by KovDimaY** [comment]
Wow, Kolyan! Thank you so so much! I do not know why I have missed this piece of docs when I was trying to find a solution to my problem!!! Thank you!!! :D :D :D 

I will try the solution on your branch after work and if everything works - I will merge it! Thank you! 👍

**Comment by KovDimaY** [comment]
Great, Kolyan! I just checked - everything works perfectly!!! So cool! Thank you so much! I am merging this! Great job!

File: index.html [hunk 1]
```diff
@@ -4,19 +4,19 @@
     <meta charset="utf-8">
     <meta name="viewport" content="width=device-width, initial-scale=0.5">
     <title>React Highcharts</title>
-    <link rel="icon" href="./favicon.ico" type="image/x-icon">
+    <link rel="icon" href="/favicon.ico" type="image/x-icon">
```
Comment (KovDimaY) [inline index.html]: Shit, to be honest - I always was thinking that "./" and "/" - it is just the same... )))) Ahah, I am so stupid )))

File: src/routes.js [hunk 2]
```diff
@@ -15,14 +15,14 @@ import NotFound from './components/notFound';
 
 // Routes definition
 export default (
-  <Router path='/' component={App}>
+  <Route path='/' component={App}>
```
Comment (KovDimaY) [inline src/routes.js]: Yes, it totally makes sense... I am going to change this in other my projects! Thanks! :D

File: index.html (hunk 1 above)
Comment (NikolayYakovenko) [reply index.html]: To be honest, I still confused with the relative and the absolute path. And sometimes I need to read documentation :)

**Pull request merged**